test/libponyc-run/regression-3874-case-1/main.pony

```diff
@@ -0,0 +1,14 @@
+actor Main
```

Comment:
Neither regression really does anything in CI at the moment because there's only a single runner run and it's in release mode, not debug mode, so these would get inlined away.